README.md

```diff
@@ -70,7 +70,6 @@ API docs for pinned cardano-wallet version: https://input-output-hk.github.io/ca
    ```
    and then add the following lines:
    ```
-   sandbox = true
    extra-sandbox-paths = /System/Library/Frameworks
```

Comment:
if the sandbox isnt enabled, `extra-sandbox-paths` has no effect and could also be removed
an empty `trusted-substituters` is also pointless and can likely be removed

Reply:
@cleverca22 I have fixed that in your new PR: https://github.com/input-output-hk/daedalus/pull/1646/commits/5cdebc6f71cbadbd70add993ad4c6cf000ea388a